boardStupid.php

```diff
@@ -19,14 +14,18 @@
         <title>boardStupid</title>
     </head>
     <body>
-    <form method="post">
+    <form action="bs_update_db.php" method="post">
         <label>Game:<br /> <input type="text" name="name"/></label><br />
         <label>Rating:<br /> <input type="text" name="rating"/></label><br />
         <label>Duration:<br /> <input type="text" name="playing_time"/></label><br />
         <label>Minimum Players:<br /> <input type="text" name="min_players"/></label><br />
         <label>Maximum Players:<br /> <input type="text" name="max_players"</label><br />
-            <input type="submit"/>
+            <input type="submit" value="Add Game"/>
         </form>
+    <form action="game_function.php" method="delete">
```

Comment:
i think delete method is just a subcategory of post and is actually a post method?